server/app/views/admin/programs/ProgramIndexViewV2.java

```diff
@@ -137,81 +155,168 @@ public ProgramDefinition getDisplayProgram(
+      List<Tag> draftRowExtraActions = Lists.newArrayList();
+      draftRowActions.add(renderEditLink(false, draftProgram.get(), request));
+      draftRowExtraActions.add(renderManageProgramAdminsLink(draftProgram.get()));
+      draftRowExtraActions.add(renderManageTranslationsLink(draftProgram.get()));
+      draftRow = renderProgramRow(false, draftProgram.get(), draftRowActions, draftRowExtraActions);
+    }
+
+    Tag activeRow = null;
+    if (activeProgram.isPresent()) {
+      List<Tag> activeRowActions = Lists.newArrayList();
+      List<Tag> activeRowExtraActions = Lists.newArrayList();
+      activeRowActions.add(maybeRenderViewApplicationsLink(activeProgram.get(), profile.get()));
+      if (!draftProgram.isPresent()) {
+        activeRowActions.add(renderEditLink(true, activeProgram.get(), request));
+        activeRowExtraActions.add(renderManageProgramAdminsLink(activeProgram.get()));
+      }
+      activeRowActions.add(renderViewLink(activeProgram.get()));
+      activeRow =
+          renderProgramRow(
+              true,
+              activeProgram.get(),
+              activeRowActions,
+              activeRowExtraActions,
+              draftProgram.isPresent() ? Styles.BORDER_T : "");
+    }
+
+    Tag titleAndStatus =
+        div()
+            .withClass(Styles.FLEX)
             .with(
-                input()
-                    .withValue(
-                        baseUrl
-                            + controllers.applicant.routes.RedirectController.programByName(
-                                    displayProgram.slug())
-                                .url())
-                    .attr("disabled", "readonly")
-                    .withClasses(Styles.W_FULL, Styles.MB_2)
-                    .withType("text"));
-
-    Tag bottomContent =
-        div(
-                p(lastEditText).withClasses(Styles.TEXT_GRAY_700, Styles.ITALIC),
-                p().withClasses(Styles.FLEX_GROW),
-                maybeRenderManageTranslationsLink(draftProgram),
-                maybeRenderEditLink(draftProgram, activeProgram, request),
-                maybeRenderViewApplicationsLink(activeProgram, profile),
-                renderManageProgramAdminsLink(draftProgram, activeProgram))
-            .withClasses(Styles.FLEX, Styles.TEXT_SM, Styles.W_FULL);
-
-    Tag innerDiv =
-        div(topContent, midContent, programDeepLink, bottomContent)
-            .withClasses(
-                Styles.BORDER, Styles.BORDER_GRAY_300, Styles.BG_WHITE, Styles.ROUNDED, Styles.P_4);
+                p(programTitleText)
+                    .withClasses(
+                        ReferenceClasses.ADMIN_PROGRAM_CARD_TITLE,
+                        Styles.W_1_4,
+                        Styles.PY_7,
+                        Styles.TEXT_BLACK,
+                        Styles.FONT_BOLD,
+                        Styles.TEXT_XL),
+                div().withClass(Styles.FLEX_GROW).with(draftRow, activeRow));
```

Comment:
suggestion (blocking): to avoid `null` values could assign this div higher up and call `with` at the end of the respective `if` statements for building these rows

Reply:
lgtm